package.json

```diff
@@ -128,6 +128,10 @@
     "@ethersproject/providers": "^5.3.1",
     "@ethersproject/strings": "^5.3.0",
     "@ethersproject/units": "^5.5.0",
+    "@fortawesome/fontawesome-svg-core": "^6.1.1",
+    "@fortawesome/free-solid-svg-icons": "^6.1.1",
+    "@fortawesome/react-fontawesome": "^0.1.18",
+    "@nomspace/nomspace": "^1.0.6",
```

Comment:
why was this added?

Reply:
please look at this

Reply:
This is used for showing fontawesome icons for filtering and sorting in farms page